Sofascore/Sofascore/ViewControllers/MainViewController.swift

```diff
@@ -8,13 +8,52 @@ class MainViewController: UIViewController {
     private let tabView: CustomTabView = .init()
     private let containerView: UIView = .init()
     private var currentEventViewController: EventViewController?
+    private let completionHandlerClient = CompletionHandlerClient()
+    private let alamofireClient = AlamofireClient()
```

Comment:
Ovdje bi iskoristila priliku da objasnim zasto smo uvodili sve te protokole. CompletionHandlerApiClient i AlamofireApiClient mozemo gledati kroz iste "naocale", a te naocale su zapravo ApiClientProtocol. Ovdje bi mogao imati samo varijablu tipa tog protokola u koju mozes ovisno o potrebi, staviti apiclient koji ti treba ovisno o situaciji. To je zapravo bila neka poanta zasto smo dodavali te protokole. Da viewcontrollera izvana zna samo da ima tu metodu getEvent i da nju treba koristiti, a sad tko to i na koji nacin izvrsava ispod haube, bas ga briga.